apps/wallet-mobile/src/features/Transactions/useCases/TxHistory/ActionsBanner.tsx

```diff
@@ -22,7 +22,7 @@ import {useSelectedWallet} from '../../../WalletManager/common/hooks/useSelected
 import {useWalletManager} from '../../../WalletManager/context/WalletManagerProvider'
 import {useStrings} from '../../common/strings'
 
-export const ActionsBanner = ({disabled = false}: {disabled: boolean}) => {
+export const ActionsBanner = () => {
```

Comment:
disabled was added for the first load, since now it doesnt open the wallet before a proper sync (exception for dev build), it is no longer needed.

Reply:
is it part of tx loading state?

Reply:
I see two things here, 1st possible a clash with another modal if the user open whatever right after loading the UI, but this would happen anyway if any other action has a modal, 2nd all pools were already migrated, so no worries (for now), the 1st issue will be solved only after building a queue for the modal.